00-HTML-CSS-basics/css 2.5/mock.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="mock.css">
+    </head>
+    <body size="cover">
+        <div class="container" align="center">
+            <header id="header" class="myThings">Buenos Aires</header>
```

Comment:
For better readability use my-things instead of camelcased myThings when naming css classes